oci/auth/azure/auth_test.go

```diff
@@ -113,6 +114,25 @@ func TestValidHost(t *testing.T) {
 	}
 }
 
+func TestGetCloudConfiguration(t *testing.T) {
```

Comment:
```suggestion
func Test_getCloudConfiguration(t *testing.T) {
```

Reply:
I don't see this naming convention in this test file, should every test be updated like so @darkowlzz ?